src/components/DataBar/DataBar.stories.tsx

```diff
@@ -62,70 +62,75 @@ export const NoSegments: StoryObj<Args> = {
   },
 };
 
-const Interactive = () => {
```

Comment:
These interactive components have to live within the render function for the code to show properly